### ISSUE TITLE
Missing FOV column in proseg output files

### ISSUE THREAD

**Issue by PietroAndrei** [dcjones/proseg#6]
Hi,

would it be possible to keep the FOV assignment for each cell (or transcript) as part of the main proseg output files? 

I am working with a relatively large CosMx dataset (2 slides, 4 samples, 386 total FOVs), so I decided to split my transcript files into chunks covering ~10 FOVs each ( ~ 10^6 transcripts/FOV), then run proseg on each chunk separately. Without the information available about the FOV of origin, it becomes a bit complicated to reassign cells to the right sample.

Thanks :)

Pietro

**Comment by dcjones** [comment]
Hi Pietro,

I just published a new version (1.0.4) that will include fov labels in the transcript and cell metadata tables.

**Comment by PietroAndrei** [comment]
Great, thanks!

I also noticed that after running the julia script to merge CosMx transcripts across FOVs, the global y coordinates are somehow flipped compared to what you should expect to find.
I adjusted the y axis orientation by adding this line at the end of the compute_global_positions() function:

```

df[:,:y] = maximum(df[:,:y]) .- df[:,:y] .+ minimum(df[:,:y]);

```


Pietro
